node-types.d.ts

```diff
@@ -88,4 +88,9 @@ declare global {
 	// by `vite`. Therefore, define it as an empty interface.
 	// eslint-disable-next-line @typescript-eslint/no-empty-interface
 	interface Worker {}
+
+	// `MessagePort` has been global since Node 15, but isn't included in
+	// `@types/node`. This is required by `undici`'s types.
```

Comment:
Is there an issue open for this on `@types/node`? If not, maybe we should open one?